terraform/main.tf

```diff
@@ -10,11 +10,23 @@ resource "layer0_load_balancer" "consul" {
   }
 
   port {
+    host_port      = 8300
+    container_port = 8300
+    protocol       = "tcp"
+  }
+
+  port {
     host_port      = 8301
     container_port = 8301
     protocol       = "tcp"
   }
 
+  port {
```

Comment:
Looks like you also need an entry here for 8302/UDP

Reply:
It doesn't look like we can specify UDP on an ELB.

Reply:
Ahh that's right. Disregard.